src/rust/dbv2/migrations/2023-10-10-171102_drop_update_orders_triggers/down.sql

```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```

Comment:
This should reinstate the triggers that are dropped at the beginning of `up.sql` , because when running migrations in reverse the original `down.sql` that created the now-defunct triggers will try to drop something that doesn't exist

Can you also run the migrations calls and make sure you can revert all the migrations too?

I believe if you had done that in the description it would've caught this

Reply:
Also, how about a comment on the original `notify_updated_order()` declaration that says how it was created improperly (as we just discovered)?

Something to the effect of (note that market/swap functions are crate improperly but are reverted in a future migration)